packages/devtools_app/test/timeline_flame_chart_test.dart

```diff
@@ -70,7 +71,7 @@ void main() {
       await tester.pumpAndSettle();
       expect(find.byType(TimelineFlameChart), findsNothing);
       expect(
-        find.byKey(TimelineScreen.emptyTimelineKey),
+        find.byKey(TimelineFlameChartContainer.emptyTimelineKey),
         findsOneWidget,
```

Comment:
nit: remove trailing comma for consistency or add trailing comma for all other `emptyTimelineKey` tests.

Reply:
doen